packages/augur-tools/src/flash/create-canned-markets-and-orders.ts

```diff
@@ -139,6 +140,7 @@ export async function createCannedMarkets(person: ContractAPI, faucet=true): Pro
     const market = await createCannedMarket(person, can, faucet);
     markets.push(market);
   }
+  markets.concat(createTemplatedMarkets(person, faucet));
```

Comment:
Array.concat does not alter the array. Maybe use push?
```suggestion
  markets.push(createTemplatedMarkets(person, faucet));
```

Reply:
nice one